locales/en/index.yml

```diff
@@ -216,6 +216,8 @@ authentication:
     cie:
       noNfcConnectionTitle: Are you sure you put the card down?
       noNfcConnectionContent: !include cie/noNfcConnection.md
+      cardRemovedEarlyTitle: You have removed the card too soon
+      cardRemovedEarlyContent: Hold the card on your smartphone until a confirmation appears on the screen. It will take a few seconds.
```

Comment:
```suggestion
      cardRemovedEarlyContent: hold the card as close as possible to the back of the device until a confirmation appears on the screen. It will take a few seconds.
```

Reply:
Ok